### PR TITLE
Skeleton1:Second Microverse project

### PR DESCRIPTION
**Pull request by Tumworobere**
**In this project, I achieved the following:**

- I built the first page of my portfolio(Skeleton).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header class="site-header">
+        <a href="#"> Tumworobere Annah</a>
+        <nav class="nav-menu">
+            <img src="./images/Vector.png" alt="Vector image ">
```
Comment (Alexoid1) [inline index.html:14]: - [ ] Use a button tag that wraps the icon.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        </nav>
+   </header>
+   <main>
+    <section class="container">
```
Comment (Alexoid1) [inline index.html:18]: - [ ] Use background color for the section container same as the Figma template.
Captura de pantalla_2021-09-21_19-27-06

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="icon">
+            <ul>
+                <li><a href="#"><img src="./images/img1.png" alt=""></a></li>
+                <li><a href="#"><img src="./images/img2.png" alt=""></a></li>
+                <li><a href="#"><img src="./images/img3.png" alt=""></a></li>
+                <li><a href="#"><img src="./images/img4.png" alt=""></a></li>
+                <li><a href="#"><img src="./images/img5.png" alt=""></a></li>
+            </ul>
```
Comment (Alexoid1) [inline index.html:37]: - [ ] Add a value to the `alt` property describing what that image is about.

**Review by Alexoid1 — CHANGES_REQUESTED** [review]

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
# Hi @Tumworobere ,

Impressive project!
You are doing a great job so far, and the project is looking good.

## Required Changes :recycle:
For the project to get approved, also implement the changes mentioned above.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!



Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (Tumworobere) [reply index.html:18]: Thank you @Alexoid1  for taking your time to review my project. I am going to work on the requested changes

**Comment by Tumworobere** [comment]
I have worked on the requested changes.
https://github.com/Tumworobere/Portfolio/pull/1

**Review by carlos-ssh — APPROVED** [review]
Hi @Tumworobere , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**